docs/pages/eas-update/estimate-bandwidth.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Estimate bandwidth usage
+description: Learn how to estimate bandwidth usage with EAS Update.
+---
+
+# Understanding Update Bandwidth Usage
+
+EAS Update enables an app to update its own non-native pieces (such as JS, styling, and images) over-the-air. This guide explains how bandwidth is consumed and optimized.
+
+## Bandwidth Calculation Breakdown
+
+Each billing cycle includes a predefined bandwidth allocation, which translates to a specific number of updates your users can download. Here's how we estimate bandwidth usage:
+
+- **Update Size:** The key factor in bandwidth consumption is the size of each update. For example, if an update modifies only the JavaScript portion of your app and is **10 MB uncompressed**, we can further optimize this with compression.
```

Comment:
```suggestion
- **Update Size:** The key factor in bandwidth consumption is the size of each update's assets not already present on device. For example, if an update modifies only the JavaScript portion of your app, when the user fetches your update it will download only the new JavaScript. The begin an example, let's say the JavaScript portion is **10 MB uncompressed**.
```